creator-node/Dockerfile

```diff
@@ -25,6 +25,20 @@ RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.10/main' >> /etc/apk/repositor
     apk add postgresql-contrib=11.12-r0 && \
     apk add postgresql-libs=11.12-r0
 
+# Install openresty with public key
+RUN echo 'http://mirror.leaseweb.com/alpine/v3.13/community' >> /etc/apk/repositories && \
+    apk add wget && \
+    wget 'http://openresty.org/package/admin@openresty.com-5ea678a6.rsa.pub' && \
```

Comment:
are we pinning openresty version here?

Reply:
Yes, because the alpine repositories are pinned

Reply:
thanks!